roles/consul/tasks/main.yml

```diff
@@ -52,55 +52,43 @@
         )
       }}
 
-
-- block:
-
-  - name: Ensure kubernetes python client is available
-    ansible.builtin.pip:
-      state: present
-      name: kubernetes
-
-  - name: Ensure Consul ServiceMonitor is present on cluster
-    kubernetes.core.k8s:
-      state: present
-      definition:
-        apiVersion: monitoring.coreos.com/v1
-        kind: ServiceMonitor
-        metadata:
-          labels:
-            release: kube-prometheus-stack
-          name: consul
-          namespace: azimuth
-        spec:
-          endpoints:
+- name: Install ServiceMonitor for Consul metrics
+  command: kubectl apply -f -
+  args:
+    stdin: "{{ consul_servicemonitor_definition | to_nice_yaml }}"
+  vars:
+    consul_servicemonitor_definition:
+      apiVersion: monitoring.coreos.com/v1
+      kind: ServiceMonitor
+      metadata:
+        name: "{{ consul_release_name }}"
+        namespace: "{{ consul_release_namespace }}"
+      spec:
+        endpoints:
           - port: http
             path: /v1/agent/metrics
-          namespaceSelector:
-            matchNames:
-            - azimuth
-          selector:
-            matchLabels:
-              # Labels should match cluster's consul-server service
-              app.kubernetes.io/managed-by: Helm
-              app: consul
-              chart: consul-helm
-              component: server
-              heritage: Helm
-              release: consul
-
-  - name: Ensure ConfigMap containing Grafana dashboard is present
-    kubernetes.core.k8s:
-      state: present
-      definition:
-        kind: ConfigMap
-        metadata:
-          labels:
-            grafana_dashboard: "1"
-            release: kube-prometheus-stack
-          name: consul-grafana-dashboard
-          namespace: azimuth
-        data:
-          consul_dashboard.json:
-            "{{ lookup('file', 'grafana_dashboard.json' ) | from_json | to_nice_json }}"
+        namespaceSelector:
+          matchNames:
+            - "{{ consul_release_namespace }}"
+        selector:
+          matchLabels:
+            app: "{{ consul_release_name }}"
+            release: "{{ consul_release_name }}"
+            component: server
 
-  when: azimuth_management_cluster_monitoring_enabled
+- name: Install Grafana dashboard for Consul metrics
+  command: kubectl apply -f -
+  args:
+    stdin: "{{ consul_dashboard_definition | to_nice_yaml }}"
+  vars:
+    consul_dashboard_definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: consul-grafana-dashboard
+        namespace: "{{ consul_release_namespace }}"
+        labels:
+          grafana_dashboard: "1"
+      data:
+        consul_dashboard.json: >-
```

Comment:
Have you tested that this dashboard renders properly in Grafana with the `>-` operator included? I seem to remember having some issues with this before.

Reply:
It seems to